container/secrets.go

```diff
@@ -81,5 +81,13 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	baseSecrets, err := readAll("/usr/share/rhel/secrets", "")
+	if err != nil {
+		return nil, err
+	}
+	overrideSecrets, err := readAll("/etc/container/rhel/secrets", "")
+	if err != nil {
+		return nil, err
+	}
+	return append(baseSecrets, overrideSecrets...), nil
```

Comment:
yes it works. If both of the directories exists, then in the container rootfs, you will eventually have the union of files in PATH1 and PATH2. With `/etc/container/rhel/secrets` overriding `/usr/share/rhel/secrets`.